README.md

```diff
@@ -1,12 +1,31 @@
-# openSUSE and Tini image
+# openSUSE Base image
 
-This project builds an openSUSE-based image which includes Tini. It can be used as a base image by projects which require Tini.
+This project builds an openSUSE-based image which includes Tini, cURL and the PostgreSQL client.
 
 ### Tini
 [Tini](https://github.com/krallin/tini) is pre-installed in the container.  If the image entrypoint is not overwritten then it will be automatically used.
 
+### PostgreSQL Client
+[PostgreSQL](https://wiki.postgresql.org/wiki/Main_Page) Client is pre-installed in the container. This enables projects to automatically install required PostgreSQL databases if required.
```

Comment:
Link to https://www.postgresql.org/ or to the actual client page (i.e. https://www.postgresql.org/docs/current/static/app-psql.html) rather than to the wiki.

Instead of saying "This enables projects to automatically install required PostgreSQL databases if required.", use the description from the client page (i.e. "psql is a terminal-based front-end to PostgreSQL. It enables you to type in queries interactively, issue them to PostgreSQL, and see the query results. Alternatively, input can be from a file or from command line arguments. In addition, psql provides a number of meta-commands and various shell-like features to facilitate writing scripts and automating a wide variety of tasks.")